{
    "id": 2486560783,
    "number": 1381,
    "title": "Failure to reconnect after suspend (permanent black window)",
    "body": "**Describe the bug**\r\n\r\nI run and use the application normally. Sometimes I close laptop lid and I go elsewhere. Laptop suspends to ram. When I open the laptop lid teams-for-linux reconnects correctly, but sometimes (like this time) it does not reconnect.\r\n\r\n**To Reproduce**\r\nSteps to reproduce the behavior:\r\n1. Run application\r\n2. Wait for connection and load.\r\n3. Suspend to ram\r\n4. Open and reconnect in a different network (mobile data or other WLAN)\r\n5. See error\r\n\r\n**Expected behavior**\r\nAfter wake up from suspend to ram, the application should reconnect normally.\r\n\r\n**Screenshots**\r\n\r\n![t4l_waiting_for_network](https://github.com/user-attachments/assets/09d611bf-0b9c-4fb4-8f79-8698362d5678)\r\n\r\n**Desktop (please complete the following information):**\r\n - OS: Debian bookwork\r\n - Installation package targz (tweaked for correct integration with plasma desktop)\r\n - Version 1.9.1\r\n\r\n**Debug**\r\n\r\n```\r\n12:11:28.963 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'\r\n12:13:03.167 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:03.172 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:04.183 › Network test failed with method https\r\n12:13:04.184 › Waiting for network...\r\n12:13:04.185 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:04.187 › Network test failed with method https\r\n12:13:04.188 › Waiting for network...\r\n12:13:04.188 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:05.196 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:05.198 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:06.210 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:06.213 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:07.221 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:07.222 › Testing network using net.request() for https://teams.microsoft.com/v2/\r\n12:13:07.425 › Network test successful with method https\r\n12:13:07.431 › Network test successful with method https\r\n<RED>12:13:09.495 › assignOnDidFailLoadEventHandler : {} - -21 - \r\n12:13:09.527 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'locked'\r\n<Problem arises>\r\n12:13:19.534 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'\r\n12:13:29.536 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'\r\n12:13:39.538 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'\r\n```\r\n\r\n**Additional context**\r\n\r\nIt looks like the following line, in red, as an error I assume:\r\n```\r\n12:13:09.495 › assignOnDidFailLoadEventHandler : {} - -21 - \r\n```\r\nmay point out the problem\r\n\r\nThanks for your continuous effort on this application.",
    "state": "closed",
    "created_at": "2024-08-26T10:49:28+00:00",
    "updated_at": "2024-10-10T07:00:54+00:00",
    "closed_at": "2024-10-10T07:00:54+00:00",
    "user": "raul-klg",
    "assignee": "IsmaelMartinez",
    "labels": [
        "known issue/workaround"
    ],
    "comments_count": 12,
    "html_url": "https://github.com/IsmaelMartinez/teams-for-linux/issues/1381",
    "is_pull_request": false
}